spock/plugins/helpers/entities.py

```diff
@@ -134,7 +134,7 @@ def handle_spawn_player(self, event, packet):
         self.ec.entities[packet.data['eid']] = entity
         self.ec.players[packet.data['eid']] = entity
         self.event.emit('entity_spawn', {'entity': entity})
-        self.event.emit('player_spawn', entity)
+        self.event.emit('entity_player_spawn', entity)
```

Comment:
I would prefer to make the event prefixes the same as the plugin name.
`entities_player_spawn`